jb/itch5/array_based_order_book.hpp

```diff
@@ -22,14 +21,16 @@ namespace itch5 {
 
 // forward declaration of
 void validate_add_order_params(int, price4_t px = price4_t(0));
+void handle_exception(std::string, std::size_t, price4_t, int);
+void handle_exception(
+    std::string, std::size_t, std::size_t, std::size_t, price4_t, int);
 
 namespace defaults {
 
 #ifndef JB_ARRAY_DEFAULTS_max_size
 #define JB_ARRAY_DEFAULTS_max_size 10000
 #endif
-
-std::size_t max_size = JB_ARRAY_DEFAULTS_max_size;
+static std::size_t Max_Size = JB_ARRAY_DEFAULTS_max_size;
```

Comment:
Please do not Camel_Case names.  Let's use max_size for that variable.  Also, it should be std::size_t const, not static.